src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -239,24 +243,26 @@ public int compare(FeedbackQuestionAttributes q1, FeedbackQuestionAttributes q2)
     }
     
     /**
-     * Gets a {@code List} of all questions for the list of questions that an
+     * Gets a {@code List} of all questions that an
      * instructor who is the creator of the course can view/submit
```

Comment:
Actually, ~~an instructor who is~~ can also be removed. Then the comment may be short enough to fit in a line.